nautobot/ipam/tests/test_api.py

```diff
@@ -456,6 +456,8 @@ def _threaded_post(self, url, data):
 class IPAddressTest(APIViewTestCases.APIViewTestCase):
```

Comment:
Should `create_data`/`update_data`/`bulk_update_data` include values for `type`?